closed/custom/modules/java.base/Copy.gmk

```diff
@@ -167,7 +167,10 @@ $(call openj9_copy_files,, \
 
 $(call openj9_copy_files,, \
 	$(OPENJ9_TOPDIR)/LICENSE \
-	$(LEGAL_DST_DIR)/openj9-notices)
+	$(LEGAL_DST_DIR)/openj9-licenses)
+$(call openj9_copy_files,, \
+	$(OPENJ9_TOPDIR)/NOTICES.md \
+	$(LEGAL_DST_DIR)/openj9-notices.md)
```

Comment:
While you're here, could you please fix the indentation of lines 178,179 to use tabs like other uses of `openj9_copy_files`?

Reply:
Updated